Ovation.FasterQC.Net.csproj

```diff
@@ -5,4 +5,9 @@
     <TargetFramework>net6.0</TargetFramework>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="CommandLineParser" Version="2.8.0" />
```

Comment:
have you tried the new one from msft yet?

Reply:
no, been using this one for a while and pretty happy with it.  🤷🏻‍♂️